api/azure-pipelines.ts

```diff
@@ -1,42 +1,130 @@
-import cheerio from 'cheerio'
 import got from '../libs/got'
 import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+import { isNullOrUndefined } from 'util'
 
 export default createBadgenHandler({
   title: 'Azure Piplines',
   examples: {
-    '/azure-pipelines/yarnpkg/yarn/Yarn Acceptance Tests': 'build',
-    '/azure-pipelines/yarnpkg/yarn/Yarn Acceptance Tests/azure-pipelines': 'build (branch)',
+    '/azure-pipelines/build/status/yarnpkg/yarn/Yarn Acceptance Tests': 'build',
+    '/azure-pipelines/build/status/yarnpkg/yarn/Yarn Acceptance Tests/azure-pipelines': 'build (branch)',
   },
   handlers: {
-    '/azure-pipelines/:org/:project/:definition/:branch?': handler
+    '/azure-pipelines/:org/:project/:definition/:branch?': buildStatus,
```

Comment:
This better be placed last as a deprecated fallback

Reply:
Done